### DISCUSSION TITLE
Directus v9.0.0-rc.19 build is not working

### DISCUSSION THREAD

**Discussion by anand-kulk** [directus/directus#3264] [Archive]
Hello there, 
 When I run the project, it doesn't pull up the admin/login page. it gets 401 on auth/refresh and 503 on /server/info
Please see attached log, any help appreciated - 

![Screen Shot 2020-12-01 at 10 41 51 AM](https://user-images.githubusercontent.com/70729249/100817869-ccc2e800-33fd-11eb-913e-f352ae8d80e0.png)

**Comment by benhaynes** [comment]
@rijkvanzanten shouldn't that 503 error in the console be red with a different icon?

**Reply by rijkvanzanten** [reply]
Kinda yes, kinda no. The logs there show the status of the request, and a request that returns a 503 status is technically a request that succeeded. It should log the cause of the 503 though

**Reply by anand-kulk** [reply]
@rijkvanzanten @benhaynes Are you planning to log the cause of error OR is there a place where I can see the log myself? Just thinking about how to proceed now...

**Reply by rijkvanzanten** [reply]
Try opening /server/info in the browser, it should print an error there 👍

**Reply by anand-kulk** [reply]
This is what it says @rijkvanzanten 

{"errors":[{"message":"Collection \"pg_stat_statements\" doesn't have a primary key column","extensions":{"service":"database","table":"pg_stat_statements","code":"SERVICE_UNAVAILABLE"}}]}

**Reply by rijkvanzanten** [reply]
See https://github.com/directus/directus/issues/3251

**Reply by anand-kulk** [reply]
Thanks @rijkvanzanten, I'll wait for updates on that thread.

**Reply by anand-kulk** [reply]
Thanks @rijkvanzanten this is now fixed, I am on rc26.
